cli.go

```diff
@@ -387,16 +388,29 @@ func (c *CLI) helpCommands(prefix string) map[string]CommandFactory {
 
 func (c *CLI) processArgs() {
 	for i, arg := range c.Args {
-		if c.subcommand == "" {
-			// Check for version and help flags if not in a subcommand
-			if arg == "-v" || arg == "-version" || arg == "--version" {
-				c.isVersion = true
-				continue
-			}
+		if arg == "--" {
+			c.isPostDashes = true
+			continue
```

Comment:
We probably just `break` here. I think the behavior of most Unix utilities is that all command-line parsing _stops_ after it sees a `--`.